coralproject/talk
# [CORL-2375] Restrict Domain Bans to >= Org Mods

## What does this PR do?
This PR fixes a bug in which Site Moderators can see the option to create a "quick domain ban" when updating a user's ban status on the frontend, even though they are barred from doing so on the backend.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

## How do I test this PR?
1. As a Org Mod and an Admin, navigate to the community view
2. Find a user who is not currently banned, and who's email domain is not currently moderated, and open the Manage Ban modal
3. Observe that the checkbox for creating a domain ban for that user's email is visible and can be selected.
4. As a Site Moderator, do the same thing but observe that this option is not visible.
 
## How do we deploy this PR?
No special considerations should be needed.
